.github/ISSUE_TEMPLATE/---relman-checklist.md

```diff
@@ -50,6 +52,7 @@ _Think something can be automated? Add it to [this doc](https://docs.google.com/
   - Merge to master
 - [ ] Cut release branch `releases/vX.X`
 - [ ] Create GitHub release build `vX.X`([instructions](https://github.com/mozilla-mobile/firefox-tv/wiki/Release-Management-Checklist#final-builds))
+- [ ] Wait for automation to email `firefox-tv@mozilla.com`, signalling that the build is done
```

Comment:
Perhaps we should mention the build is automatically uploaded here?

Reply:
Good point!